### ISSUE TITLE
Turn rain into a const

### ISSUE THREAD

**Issue by sbromberger** [leonistor/raindrops-benchmark#1]
```
const rain = [(3,"Pling"), (5,"Plang"), (7,"Plong")]
```
and then run the benchmarks. I get

```
FUNCTIONAL
BenchmarkTools.Trial: 
  memory estimate:  6.42 KiB
  allocs estimate:  167
  --------------
  minimum time:     6.783 μs (0.00% GC)
  median time:      6.895 μs (0.00% GC)
  mean time:        8.560 μs (15.03% GC)
  maximum time:     8.222 ms (99.88% GC)
  --------------
  samples:          10000
  evals/sample:     5

IMPERATIVE
BenchmarkTools.Trial: 
  memory estimate:  416 bytes
  allocs estimate:  7
  --------------
  minimum time:     2.225 μs (0.00% GC)
  median time:      2.255 μs (0.00% GC)
  mean time:        2.389 μs (4.72% GC)
  maximum time:     1.133 ms (99.62% GC)
  --------------
  samples:          10000
  evals/sample:     9

RATIO
BenchmarkTools.TrialRatio: 
  time:             3.0572864321608035
  gctime:           1.0
  memory:           15.807692307692308
  allocs:           23.857142857142858
```

**Comment by leonistor** [comment]
hmmm... :astonished: :confused: 
With `const rain` the results are similar even after taking out the shuffling.
Any hints?

**Comment by sbromberger** [comment]
It's because you're benchmarking in global scope, which is a no-no. Put `rain` inside the functions and you should see similar improvements.

**Comment by leonistor** [comment]
Corrected benchmarks: 933ea22d1b90e580e715e9d7513c25877f6519ca
